packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.ts

```diff
@@ -109,65 +110,72 @@ export class CryptoWatchPriceFeed extends PriceFeedInterface {
       );
 
-      const format = (value: BN | null | undefined) => {
-        const unformattedValue = value && (this.invertPrice ? this._invertPriceSafely(value) : value);
-        if (!unformattedValue) return "[no value found]";
-        return formatFixed(unformattedValue.toString(), this.priceFeedDecimals);
-      };
-
-      throw new Error(`
-        Cryptowatch price feed ${this.uuid} didn't return an ohlc for time ${time}.
-        Closest price point before is close time: ${before?.closeTime} (- buffer of ${
-        this.historicalTimestampBuffer
-      } seconds), close price: ${format(before?.closePrice)}.
-        Closest price point after is open time: ${after?.openTime} (+ buffer of ${
-        this.historicalTimestampBuffer
-      } seconds), open price: ${format(after?.openPrice)}.
-        To see these prices, make a GET request to:
-        https://api.cryptowat.ch/markets/${this.exchange}/${this.pair}/ohlc?after=${
-        (before?.openTime || 0) + this.ohlcPeriod
-      }&before=${after?.closeTime}&periods=60
-      `);
-    }
+      this.logger.debug({
+        at: "CryptoWatchPriceFeed",
+        message: "Using historical timestamp buffer to match time",
+        historicalTimestampBuffer: this.historicalTimestampBuffer,
+        time: time,
+        before: before
+          ? `openTime: ${before?.openTime}, closeTime: ${
+              before?.closeTime
+            }, openPrice: ${before?.openPrice.toString()}, closePrice: ${before?.closePrice.toString()}`
+          : before,
+        after: after
+          ? `openTime: ${after?.openTime}, closeTime: ${
+              after?.closeTime
+            }, openPrice: ${after?.openPrice.toString()}, closePrice: ${after?.closePrice.toString()}`
+          : after,
+      });
 
-    const returnPrice = this.invertPrice ? this._invertPriceSafely(match.openPrice) : match.openPrice;
-    if (!returnPrice) throw new Error(`${this.uuid} -- invalid price returned`);
-    if (verbose) {
-      console.group(`\n(${this.exchange}:${this.pair}) Historical OHLC @ ${match.closeTime}`);
-      console.log(`- ✅ Open Price: ${formatFixed(returnPrice.toString(), this.priceFeedDecimals)}`);
-      console.log(
-        `- ⚠️  If you want to manually verify the specific exchange prices, you can make a GET request to: \n- https://api.cryptowat.ch/markets/${this.exchange}/${this.pair}/ohlc?after=${match.closeTime}&before=${match.closeTime}&periods=60`
-      );
-      console.log(
-        '- This will return an OHLC data packet as "result", which contains in order: \n- [CloseTime, OpenPrice, HighPrice, LowPrice, ClosePrice, Volume, QuoteVolume].'
-      );
-      console.log(
-        "- We use the OpenPrice to compute the median. Note that you might need to invert the prices for certain identifiers like USDETH."
-      );
-      console.groupEnd();
+      // If there is a `before` period, then use its close time, otherwise use the `after` period's open time.
```

Comment:
Can you explain what changed in these logs/checks?

Reply:
The `if` statement logic is: 
- If there is a `before` period, then use its `close` price, otherwise if there is an `after` period, use its `open` price. If none, throw.